libraries/AC_Fence/AC_Fence.h

```diff
@@ -116,8 +115,11 @@ class AC_Fence
     bool geofence_failed() const;
 
 private:
+
+    float curr_alt;
```

Comment:
Humm, shouldn't this follow the other variables and have an underscore prefix? Putting it next to the `home_distance` also seems a good idea.